desc/geometry/surface.py

```diff
@@ -469,6 +470,61 @@ def from_input_file(cls, path):
         )
         return surf
 
+    @classmethod
+    def from_near_axis(cls, aspect_ratio, elongation, mirror_ratio, axis_Z, NFP=1):
+        """Create a surface from a near-axis model for quasi-poloidal/quasi-isodynamic.
+
+        Parameters
+        ----------
+        aspect_ratio : float
```

Comment:
is there a formula in a paper that can be referenced here for this function?

Reply:
There is a formula, but it's unpublished. Just a PDF Matt Landreman sent me.

Reply:
Ok, maybe not necessary right now but would be good to remember if it becomes published, I like having formulas for things